python/graphstorm/eval/utils.py

```diff
@@ -213,6 +214,172 @@ def calc_dot_neg_head_score(heads, tails, num_chunks, chunk_size,
     tmp = tails.reshape(num_chunks, chunk_size, hidden_dim)
     return th.bmm(tmp, heads)
 
+def calc_rotate_pos_score(h_emb, t_emb, r_emb, rel_emb_init, gamma, device=None):
+    r""" Calculate RotatE Score for positive pairs
+
+        Score function of RotateE measures the angular distance between
+        head and tail elements. The angular distance is defined as:
+
+        .. math:
+
+            d_r(h, t)=\|h\circ r-t\|
+
+        The RotatE score function is defined as:
+
+        .. math:
+
+            gamma - \|h\circ r-t\|^2
+
+        where gamma is a margin.
+
+        For more detials please refer to https://arxiv.org/abs/1902.10197
+        or https://dglke.dgl.ai/doc/kg.html#rotatee.
+
+        Parameters
+        ----------
+        h_emb: th.Tensor
+            Head node embedding.
+        t_emb: th.Tensor
+            Tail node embedding.
+        r_emb: th.Tensor
+            Relation type embedding.
+        rel_emb_init: float
+            The initial value used to bound the relation embedding initialization.
+        gamma: float
+            The gamma value used for shifting the optimization target.
+
+        Return
+        ------
+        rotate_score: th.Tensor
+            The RotatE score.
+    """
+    if device is not None:
+        r_emb = r_emb.to(device)
+        h_emb = h_emb.to(device)
+        t_emb = t_emb.to(device)
+
+    re_head, im_head = th.chunk(h_emb, 2, dim=-1)
```

Comment:
```suggestion
    real_head, img_head = th.chunk(h_emb, 2, dim=-1)
```
Could you change all `re` to `real` and all `im` to `img` in this function, to keep consistency with other score functions?